stablehlo/reference/Api.cpp

```diff
@@ -170,19 +170,82 @@ LogicalResult removeDynamism(ModuleOp module, func::FuncOp func,
   return success();
 }
 
+bool isQuantizedTypes(TypeRange types) {
+  return llvm::any_of(types, [](Type type) {
```

Comment:
Should this be `all_of`? To me the name reads like "are all of these types quantized".

Reply:
renamed it to suggest if any of the parameters is of Quantized types.